dashboard/app/models/levels/blockly.rb

```diff
@@ -300,6 +300,7 @@ def localized_blockly_level_options(script)
 
         set_unless_nil(level_options, 'longInstructions', localized_long_instructions)
         set_unless_nil(level_options, 'failureMessageOverride', localized_failure_message_override)
+        set_unless_nil(level_options, 'startHtml', localized_start_html(level_options['startHtml']))
```

Comment:
`level_options['startHtml']` is only not `nil` when the level `is_a?` AppLab, according to: https://github.com/code-dot-org/code-dot-org/blob/eb836d6c11bf50323fc4bf5b3f837656b057f6d9/dashboard/app/models/levels/blockly.rb#L376

So I'm curious if this breaks on other levels that still hit this localization path.

Reply:
thanks for the callout! I added a nil check to the localization method.

Reply:
I see your update! Looks good, yes.